coffeelint.json

```diff
@@ -18,6 +18,9 @@
     "space_operators": {
         "level": "error"
     },
+    "max_line_length": {
+        "value": "100"
```

Comment:
80 is just wayyy too low, it was causing issues in the test files. Easier to just up the limit than to adhere to a dogmatic 80.